cloud_function/main.py

```
@@ -0,0 +1,66 @@
+from datetime import datetime
+
+# Import the sync function
+from migrate_database import sync_database
+
+def log(message):
+    """Print message with timestamp"""
+    timestamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    print(f"[{timestamp}] {message}")
+
+@functions_framework.http
+def database_sync(request):
```

Ruff (0.14.1)

Unused function argument: `request`

(ARG001)